app/src/main/java/org/oppia/android/app/help/thirdparty/ThirdPartyDependencyListFragment.kt

```diff
@@ -30,8 +36,16 @@ class ThirdPartyDependencyListFragment : InjectableFragment() {
   override fun onCreateView(
     inflater: LayoutInflater,
     container: ViewGroup?,
-    savedInstanceState: Bundle?
+    savedInstanceState: Bundle?,
```

Comment:
The `,` at the ned needs to be removed right?

Reply:
Removed.